src/Controllers/LoginController.php

```diff
@@ -55,10 +63,29 @@ public function loginAction()
         return redirect()->to(config('Auth')->loginRedirect())->withCookies();
     }
 
+    /**
+     * Returns the rules that should be used for validation.
+     *
+     * @return string[]
+     */
+    protected function getValidationRules(): array
+    {
+        $rules = [
+            'email'    => 'required',
```

Comment:
```suggestion
            'email'    => 'required|max_length[254]|valid_email',
```

Reply:
I didn't even notice it's possible "required_without".